release-please-config.json

```diff
@@ -2,6 +2,7 @@
   "bootstrap-sha": "47035e3bd4e3ac399319b3421ce86a82f1c1ab21",
   "plugins": ["node-workspace"],
   "prerelease": true,
+  "prerelease-type": "beta",
```

Comment:
**issue:** This was released in https://github.com/googleapis/release-please/releases/tag/v16.1.0 so we will need to bump our version of release please too

Reply:
i couldn't get it working even with the new version. ivo's going to do it semi-manually